test/build.bats

```diff
@@ -510,7 +510,7 @@ OUT
   cached_tarball "jruby-1.7.9" bin/foo.exe bin/bar.dll bin/baz.bat
 
   run_inline_definition <<DEF
-install_package "jruby-1.7.9" "http://jruby.org/downloads/jruby-bin-1.7.9.tar.gz" jruby
+install_package "jruby-1.7.9" "https://s3.amazonaws.com/jruby-org/downloads/1.7.9/jruby-bin-1.7.9.tar.gz" jruby
```

Comment:
Again, I'm sad that the canonical download location for JRuby isn't under the `jruby.org` domain anymore. But I guess there's nothing we can do since they seem to be using `s3.amazonaws.com` on their official download page anyway.